spyder/widgets/projects/projectdialog.py

```diff
@@ -152,8 +152,9 @@ def _get_project_types(self):
 
     def select_location(self):
         """Select directory."""
-        location = getexistingdirectory(self, _("Select directory"),
-                                        self.location)
+        location = os.path.normcase(getexistingdirectory(self,

```

Comment:
Please use `osp.normcase` instead of `os.path.normcase`.

Reply:
Good catch @jnsebgosselin!

@vitoriahmc, that's our convention throughout our code: `import os.path as osp`.